sunpy/instr/aia/response.py

```diff
@@ -0,0 +1,308 @@
+"""
+Calculate SDO/AIA wavelength and temperature response functions.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import os
+import warnings
+from collections import namedtuple
+
+import numpy as np
+import h5py
```

Comment:
Just realized that import is actually unused. The only place that h5py needs to be is in response_utils.py where the make_emiss_table function lives. 

By not caching, I assume you mean just keeping in memory...which means recalculating each time. I don't think this is a good option, particularly since h5py is so easily installed.